### PR TITLE
Remove globals.js

### PR DESCRIPTION
**Pull request by mjethani**
This patch updates some more files to remove `src/js/globals.js` entirely.

### CODE REVIEW

File: src/js/biditrie.js [hunk 1]
```diff
@@ -19,8 +19,6 @@
     Home: https://github.com/gorhill/uBlock
 */
 
-/* globals WebAssembly vAPI */
```
Comment (mjethani) [inline src/js/biditrie.js:22]: JSHint gives a warning that this is redundant.

File: src/js/logger-ui-inspector.js [hunk 2]
```diff
@@ -47,7 +43,7 @@ if (
 
 /******************************************************************************/
 
-const logger = globals.logger;
+const logger = self.logger;
```
Comment (mjethani) [inline src/js/logger-ui-inspector.js:46]: There might be a better way to share `logger` between the two files, but let's use `self` for now.

File: src/js/biditrie.js (hunk 1 above)
Comment (gorhill) [reply src/js/biditrie.js:22]: Need a comma after `WebAssembly`.

Comment (gorhill) [reply src/js/biditrie.js:22]: Never mind, I also see no warning when removing the line.

Comment (mjethani) [reply src/js/biditrie.js:22]: Sorry, you were right. It was for ESLint too: https://github.com/gorhill/uBlock/pull/3850

**Pull request merged**